src/richie/apps/core/templates/richie/pagination.html

```diff
@@ -2,9 +2,7 @@
 
 {% if is_paginated %}
 <div class="pagination">
-  <div id="pagination-label" class="offscreen">{% trans "Pagination" %}</div>
```

Comment:
I'm thinking maybe it would be simpler and easier to understand if we take the wrapper markup out of this template and keep it in the calling template where it can be customized?

Reply:
@Leimi And you can still share the code by introducing a new template. In the page where you want the pagination you put:
```
{% include richie/pagination_wrapper_new_template with label="Custom pagination" %}
```

and in the wrapper template:

Reply:
:sweat_smile:

Reply:
Well… Yeah, what was I thinking? This solution is way simpler haha. Will update, thanks.

Reply:
@Leimi be careful I just merged this PR which adds new pagination and should also benefit from your refactoring : https://github.com/openfun/richie/pull/1580